services/frontend-service/src/assets/_variables.scss

```diff
@@ -5,6 +5,25 @@ $nav-list-item-padding: 3em;
 $nav-indicator-width: 8px;
 $nav-indicator-border-radius: 0px 5px 5px 0px;
 
+//Sidebar variables
+$sidebar-width: 30em;
+$sidebar-planned-actions-button-margin: 1em;
+$sidebar-planned-actions-title-right-centering: 9em;
+$sidebar-apply-button-height: 3em;
+$sidebar-apply-button-font-weight: 1.5em;
+$sidebar-planned-actions-border-radius: 1em 0 0 0;
+$sidebar-apply-border-radius: 0 0 0 1em;
+$sidebar-display-button-border-radius: 1em 0 0 1em;
+$sidebar-hidden-displacement-large-screen: -100vw;
+$sidebar-transition-duration-large-screen: 1.5s;
+$sidebar-hidden-displacement-medium-screen: -200vw;
+$sidebar-transition-duration-medium-screen: 2s;
+$sidebar-hidden-displacement-small-screen: -400vw;
+$sidebar-transition-duration-small-screen: 3s;
```

Comment:
for the displacement, you are still using `vw` which is a relative unit to the size of the screen. 

but this can be done in a simpler way. e.g. since we know the constant width of the sidebar, we can just shift it to the right by that exact value. and it will always be hidden on any screen size. 

I strongly suggest not handling screen sizes unless there's a very good reason to do that.

```
$sidebar-displacement: -$sidebar-width
```